### PR TITLE
Editable Footer

### PR DESCRIPTION
**Pull request by kaalleen**
This pull request should add an editable footer as suggested in #346 

Bugfix: the massive dom manipulation through javascript caused that the event handlers ignored changes made on operator detailed view pages within some contenteditable fields (title, client, order number and file picker).

### CODE REVIEW

File: print/resources/inkstitch.js [hunk 1]
```diff
@@ -397,6 +414,118 @@ $(function() {
...
+  });
+
+  $('#tb-remove').click(function() {
+    formatText('removeFormat');
+  });
+
+  $('#tb-reset').click(function() {
+    var htmlMode = getEditMode();
+    if(htmlMode) {
+      $('#footer-info-text').text('<div>Proudly generated with <a href="http://inkstitch.org/" target="_blank">Ink/Stitch</a></div>');
```
Comment (lexelby) [inline print/resources/inkstitch.js]: Unfortunately, this won't work for non-English speakers. :(

When I had other similar situations, I put the text into a hidden div (dispay: none) in the jinja2 template, which allowed me to translate the text.  Then in the javascript, I fetched it out of the div and inserted it where I actually wanted it.

Can we do that here?

File: print/templates/footer.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <footer>
     <p class="num_pages">{{ _('Page') }} <span class="page-num"></span>/<span class="total-page-num"></span></p>
-    <p>Proudly generated with <a href="http://inkstitch.org/" target="_blank">Ink/Stitch</a></p>
+    <div class="footer-info" data-field-name="footer-info">Proudly generated with <a href="http://inkstitch.org/" target="_blank">Ink/Stitch</a></div>
```
Comment (lexelby) [inline print/templates/footer.html]: Hah, looks like we never even translated this in the first place!  Oops.

File: print/templates/ui.html [hunk 3]
```diff
@@ -16,82 +16,123 @@ <h1>{{ _('Settings') }}</h1>
       
       <div id="tabs">
         <button class="tab active">{{ _('Page Setup') }}</button>
+        <button class="tab" id="operator-tab">{{ _('Operator') }}</button>
```
Comment (lexelby) [inline print/templates/ui.html]: Hmmm... maybe this tab should be called "Branding"?

**Comment by lexelby** [comment]
Wow, this is really incredible!! You've done an amazing job.  I just love how this works, and it's an extremely useful feature for folks running embroidery shops.  Great work!

I'm impressed by how small the code for the rich text editing was.  I'd forgotten about the `execCommand` function so I assumed we'd need to pull in another jquery library or something.

A few comments, but other than that, IMO this is ready to go.  As far as I'm concerned you can feel free to merge pull requests to master even if I'm not around to approve them.  Even if I have ideas for improvement, I can always make a new PR. :)  GitHub will make you get an approval from at least one person before merging, but I just did that to encourage us to get more than one set of eyes on stuff.

**Comment by kaalleen** [comment]
Fixed. Now we need someone to approve ;)

**Pull request merged**